cognite/client/_api/raw.py

```diff
@@ -470,3 +483,12 @@ def list(
                 "columns": columns,
             },
         )
+
+    def _make_columns_param(self, columns: List[str]) -> str:
+        if not isinstance(columns, List):
```

Comment:
```suggestion
    def _make_columns_param(self, columns: Optional[List[str]]) -> Optional[str]:
        if columns is None:
            return None
        if not isinstance(columns, List):
```

Reply:
And then remove the `if columns is not None` check from `list()` and `__iter__()`